src/Sylius/Behat/Context/Ui/Shop/CartContext.php

```diff
@@ -0,0 +1,414 @@
+        );
+    }
+
+    /**
+     * @Given quantity value should be :quantity
+     */
+    public function quantityValueShouldBe($quantity)
+    {
+        $this->summaryPage->open();
+
+        Assert::eq(
+            $this->summaryPage->getQuantity(),
+            $quantity,
+            'Quantity of product should be %2$s, but it is %s'
+        );
+    }
+
+    /**
+     * @Then I should be on my cart summary page
+     */
+    public function shouldBeOnMyCartSummaryPage()
+    {
+        Assert::true(
+            $this->summaryPage->isOpen(),
+            'Cart summary page should be open, but it does not.'
+        );
+    }
+
+    /**
+     * @Then I should be notified that the product has been successfully added
+     */
+    public function iShouldBeNotifiedThatItHasBeenSuccessfullyAdded()
+    {
+        $this->notificationChecker->checkNotification('Item has been added to cart.', NotificationType::success());
+    }
+
+    /**
+     * @Then I should see one item on my product list
+     */
+    public function iShouldSeeOneItemOnMyProductList()
+    {
+        Assert::true(
+            $this->summaryPage->isSingleItemOnPage(),
+            'There should be only one item on list, but it does not.'
+        );
+    }
+
+    /**
+     * @Then this item should have name :itemName
+     */
+    public function thisProductShouldHaveName($itemName)
+    {
+        Assert::true(
+            $this->summaryPage->isItemWithName($itemName),
+            sprintf('The product with name %s should appear on the list, but it does not.', $itemName)
+        );
+    }
+
+    /**
+     * @Then this item should have variant :variantName
+     */
+    public function thisItemShouldHaveVariant($variantName)
+    {
+        Assert::true(
+            $this->summaryPage->isItemWithVariant($variantName),
+            sprintf('The product with variant %s should appear on the list, but it does not.', $variantName)
+        );
+    }
+
+    /**
+     * @When /^I add (this product) with ([^"]+) "([^"]+)" to the cart$/
+     */
+    public function iAddThisProductWithSizeToTheCart(ProductInterface $product, $optionName, $optionValue)
+    {
+        $this->productShowPage->open(['slug' => $product->getSlug()]);
+
+        $option = $this->sharedStorage->get(sprintf('%s_option', $optionName));
+
+        $this->productShowPage->addToCartWithOption($option, $optionValue);
+    }
+
+    /**
+     * @Given /^(this product) should have ([^"]+) "([^"]+)"$/
+     */
+    public function thisItemShouldHaveSize(ProductInterface $product, $optionName, $optionValue)
+    {
+        Assert::contains(
+            $this->summaryPage->getProductOption($product->getName(), $optionName),
+            $optionValue,
+            'The product should have option with value %2$s, but it has option with value %s.'
+        );
+    }
+
+    /**
+     * @param string $price
+     *
+     * @return int
+     */
+    private function getPriceFromString($price)
```

Comment:
IMO we should implement this method in ``SummaryPage`` and use it in ``getItemDiscountPrice($productName), getItemRegularPrice($productName)``.